commands/tar2zip.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
```

Comment:
do I need to specify this as `python3` here  and below in the caller script?

Reply:
Is your file actually python3-specific?

Reply:
No,  it works on both version. I guess on my system there's only `python3`, so launching from the shebang doesn't work.